sinatra/sinatra_app.rb

```diff
@@ -1,14 +1,13 @@
 set :public_dir, Proc.new { File.join(root, "_site") }
 set :views, Proc.new { File.join(File.dirname(__FILE__), "views") }
 
-post '/api', :provides => :json do
-  #
-  #
-  # Code to handle api calls
-  #
-  #
-  puts "Here comes your data!"
-  puts params
+require './sinatra/skillsengine.rb'
```

Comment:
made it snake_case. i think the weird setup of the project makes 'skills_engine.rb' not work so i had to add sinatra to the path.